source/documentation/tools/create-a-derived-table/data-modelling-concepts.md

```diff
@@ -1,5 +1,390 @@
+
+## How should I style?
+
+In Analytics Engineering we have spent several months going through dbts style guide and discussing what aspects we agree and disagree on. We have taken their style guide and changed it to suit our needs. The below is that guide. Please read through and make sure you are familiar with our style guide before starting projects in create-a-derived-table.
+
+## Automation
+
+As part of our review of a dbt style guide and putting together our own, we have also worked on our own formatters and linters to help standardise our code and make it as easy as possible to follow the rules we have set out. Find the guide to using those [here](need to write a guide to using linter)
+
+
+
+# How we style our dbt models
+
+## Fields and model names
+
+- 👥 Models should be pluralized, for example, `customers`, `orders`, `products`. Alhough this is a good best practice, we accept that this may not work with the projects you are working on, so if you cannot keep to it then that is fine.
+- 🔑 Each model should have a primary key and that primary key should be the first field in the table.
+- 🔑 The primary key of a model should be named `<object>_id`, for example, `account_id`. This makes it easier to know what `id` is being referenced in downstream joined models.
+- Use underscores for naming dbt models; avoid dots or camel case.
+  - ✅  `models_without_dots`
+  - ❌ `models.with.dots`
+  - ❌ `CamelCaseModels`
+  - Most data platforms use dots to separate `database.schema.object`, so using underscores instead of dots reduces your need for [quoting](/reference/resource-properties/quoting) as well as the risk of issues in certain parts of dbt Cloud. For more background, refer to [this GitHub issue](https://github.com/dbt-labs/dbt-core/issues/3246).
+- 🔑 Keys should be string data types. Additionally we adives using the hash function `dbt_utils.generate_surrogate_key` ([guidance here](https://github.com/dbt-labs/dbt-utils?tab=readme-ov-file#generate_surrogate_key-source)) to create unique keys. This ensures there is a unique id for each row as well as making the ids uniform in length. 
+- 🔑 Consistency is key! Use the same field names across models where possible. For example, a key to the `customers` table should be named `customer_id` rather than `user_id` or 'id'.
+- ❌ Do not use abbreviations or aliases. Emphasize readability over brevity. For example, do not use `cust` for `customer` or `o` for `orders`. Again, in AE we accept that in some cases this may not be possible. We want to prioritise readability, so try your best to be as descriptive as possible, however, if this is not practical then do not lose sleep over it.
```

Comment:
A little ironic to use the abbreviation AE in a point about not abbreviating - I think it's just as clear with 'in AE' removed